suite/tests/CMakeLists.txt

```diff
@@ -2536,11 +2536,13 @@ endif ()
       "" "-thread_private -cache_bb_unit_init 4K" "")
   endif ()
 
-  if (X86) # Tests x86 reachability.
+  if (X64) # Tests x86 reachability.
```

Comment:
X64 is on for AArch64 so this is not enough.

Reply:
Already fixed.